win32ss/gdi/ntgdi/bitblt.c

```diff
@@ -7,6 +7,10 @@
  */
 
 #include <win32k.h>
+
+#define NDEBUG
+#include <debug.h>
+
```

Comment:
Can't we use the win32k debug channels here? (and thus replace the DPRINTs with these)

Reply:
@HBelusca, if you do a grep of win32ss/gdi for WINE_DEFAULT_DEBUG_CHANNEL you will see that the only places that this is used is under a "wine" subdirectory. In all of the other cases the "#include <debug.h>" is used to produce debug messages. So I followed this same pattern. Your question was "Cant' we use ..." and maybe we could, but I did not investigate this and I certainly do not have the expertise to even hazard a guess without investigation. Thanks.

Reply:
Not really, see for example: https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/ntuser/desktop.c;h=365a4e5a9a40321defc4ea9c37174b7f9785ed53;hb=HEAD#l252

Reply:
@HBelusca, please note that I specified "win32ss/gdi" and your path did not meet this criteria. Thanks.

Reply:
But I was talking about win32k debug channels, not wine debug channels ;)

Reply:
Oh, yes. Thank you for the clarification. :)

Reply:
Leave this as is for now.